src/main/java/uk/gov/justice/hmpps/prison/api/model/BedAssignment.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.justice.hmpps.prison.api.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode
+@ToString
+@Data
+public class BedAssignment {
+    private Long bookingId;
+    private Long livingUnitId;
+    private LocalDate assignmentDate;
```

Comment:
Do you need to separate the date and times into separate fields?

Reply:
They are returned like that from the mapping and we may end up using them like this. I'll probably update at some point as we've had a play.